src/pages/YourShoppingListsHere/Anton/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect, useRef } from 'react'
+import styled from 'styled-components'
+import { Button } from 'antd'
+import { PercentageOutlined } from '@ant-design/icons'
+import { stringify } from 'rc-field-form/es/useWatch'
```

Comment:
some of the imports are not used, do remove them

Terminal:
![image](https://user-images.githubusercontent.com/75428027/196076079-3354175a-f15b-4828-a313-77c65f889f90.png)